frontend/src/components/enroll/index.js

```diff
@@ -20,64 +23,35 @@ class Enroll extends Component {
       });
   }
 
+  loaded = () => {
```

Comment:
Lets not use this weird workaround and just set an explicit height via css and it will solve the issue. 

```
.enroll-iframe {
  height: 1275px;
}
```

will solve and then we can remove this method and load counter from state

Reply:
then 
```jsx
let height = this.state.loadCounter % 2 === 0 ? 1275 : 400
this.setState({iframeHeight: height, loadCounter: this.state.loadCounter += 1})
```